src/components/button/button.module.scss

```diff
@@ -0,0 +1,13 @@
+
+.request{
+    background-color: black;
+    height: 70px;
+    width: 10%;
+    border: none;
+    cursor: pointer;
+    border-radius: 0 10px 10px 0;
+  }
+  .request:hover{
```

Comment:
Something is wrong with indentation AND spaces))

Since we use SCSS, a hover selector might be nested inside:
```css
.request {
   /* base styles */

  &:hover {
     /* hover styles */
  }
}

```